core/runtime/execute_engine.cpp

```diff
@@ -117,6 +117,8 @@ void setup_input_tensors(
     auto shape = core::util::toVec(dims);
     LOG_DEBUG("Input Name: " << name << " Shape: " << dims);
 
+    void* tensor_addr = nullptr;
```

Comment:
I strongly want to avoid having nullptr basically anywhere, we should be looking for some sane default